hax/hax/util.py

```diff
@@ -896,35 +909,38 @@ def get_service_health(self, node: str, svc_id: int) -> ServiceHealth:
         # Respective values are for local and remote nodes.
         # {(consul_svc_status, motr_process_status):(local_node_ha_status,
         #                                            remote_node_ha_status)}
+
         svc_to_motr_status_map = {
-            ('passing',
-             'M0_CONF_HA_PROCESS_STARTING'): (ServiceHealth.OFFLINE,
-                                              ServiceHealth.OK),
-            ('passing',
-             'M0_CONF_HA_PROCESS_STOPPING'): (ServiceHealth.OFFLINE,
-                                              ServiceHealth.UNKNOWN),
-            ('passing',
-             'M0_CONF_HA_PROCESS_STARTED'): (ServiceHealth.OK,
-                                             ServiceHealth.OK),
-            ('passing',
-             'M0_CONF_HA_PROCESS_STOPPED'): (ServiceHealth.OFFLINE,
-                                             ServiceHealth.UNKNOWN),
-            ('passing',
-             'Unknown'): (ServiceHealth.UNKNOWN, ServiceHealth.UNKNOWN),
-            ('warning',
-             'M0_CONF_HA_PROCESS_STOPPING'): (ServiceHealth.OFFLINE,
-                                              ServiceHealth.STOPPED),
-            ('warning',
-             'M0_CONF_HA_PROCESS_STARTED'): (ServiceHealth.OFFLINE,
-                                             ServiceHealth.FAILED),
-            ('warning',
-             'M0_CONF_HA_PROCESS_STOPPED'): (ServiceHealth.STOPPED,
-                                             ServiceHealth.STOPPED),
-            ('warning',
-             'M0_CONF_HA_PROCESS_STARTING'): (ServiceHealth.OFFLINE,
-                                              ServiceHealth.OFFLINE),
-            ('warning',
-             'Unknown'): (ServiceHealth.UNKNOWN, ServiceHealth.UNKNOWN)}
+            MotrConsulProcStatus('passing', 'M0_CONF_HA_PROCESS_STARTING'):
+            MotrProcStatusLocalRemote(ServiceHealth.OFFLINE,
```

Comment:
[Optional]:
These enum names are VERY long which makes the code a bit unreadable. Please note that it could be simplified a bit by using reference variables like this:
```python
        local_remote = MotrProcStatusLocalRemote
        proc_status = MotrConsulProcStatus

        svc_to_motr_status_map = {
            proc_status('passing', 'M0_CONF_HA_PROCESS_STARTING'):
            local_remote(ServiceHealth.OFFLINE, ServiceHealth.OK),
            proc_status('passing', 'M0_CONF_HA_PROCESS_STOPPING'):
            local_remote(ServiceHealth.OFFLINE, ServiceHealth.UNKNOWN),
            proc_status('passing', 'M0_CONF_HA_PROCESS_STARTED'):
            local_remote(ServiceHealth.OK, ServiceHealth.OK),
```
and so forth.
Optional, so feel free to ignore if you don't like this.

Reply:
Addressed.